testing/PythonTests/central_control/range_test.py

```diff
@@ -0,0 +1,11 @@
+from src.central_control.scripts.range import Range
+
+
+def test_overlaps():
+    range1 = Range(1, 5)
+    range2 = Range(4, 7)
+    range3 = Range(6, 10)
+
+    assert range1.overlaps(range2)
```

Comment:
what is this assert checking for exactly?

Reply:
Pytest works by using the `assert` keyword that is similar to `assertTrue()` in JUnit. Just checking that whatever follows is true, so in this case it would check that `range1.overlaps(range2)` is `True`. You can read more [here](https://docs.pytest.org/en/7.1.x/getting-started.html#create-your-first-test)